crates/node/src/network_server/metrics.rs

```diff
@@ -277,15 +282,15 @@ pub async fn render_metrics(State(state): State<NodeCtrlHandlerState>) -> String
 
         // Properties (Gauges)
         // For properties, we need to get them for each column family.
+        labels[global_labels.len() + 1] =
+            format!("db=\"{}\"", formatting::sanitize_label_value(&db.name));
         for cf in &db.cfs() {
             let sanitized_cf_name = formatting::sanitize_label_value(cf);
-            let mut cf_labels = Vec::with_capacity(labels.len() + 1);
-            labels.clone_into(&mut cf_labels);
-            cf_labels.push(format!("cf=\"{sanitized_cf_name}\""));
+            labels[global_labels_len + 1] = format!("cf=\"{sanitized_cf_name}\"");
```

Comment:
If we replace this call with `labels.push(format!("cf=\"{sanitized_cf_name}\""))` and call `labels.pop()` at the end of the loop, then we wouldn't have to extend the vec before and explicitly specify lengths when passing the `labels` to the functions.

Reply:
I also considered implementing it in the way of using "push" and "pop" before. This can avoid the complexity of pre-calling "extend" and using indexes to obtain sub-arrays. However, I'm not quite sure which method is better.